packages/controller/src/main.ts

```diff
@@ -3005,6 +3005,21 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
             restart(() => !isStopping && stop(false));
             break;
         }
+
+        case 'sendToSentry': {
+            const message: string = msg.message.message;
+
+            const sentryInstance = pluginHandler.getPluginInstance('sentry');
+
+            if (!sentryInstance) {
+                logger.debug(`${hostLogPrefix} Do not send message "${message}" to Sentry, because it is disabled`);
+                return;
+            }
+
+            // @ts-expect-error Plugin is not well typed and SentryPlugin has no types at all currently
+            sentryInstance.getSentryObject().captureException(message);
```

Comment:
hm ... is this really bringing that much? Then we would have one event per broken object ID ... which could contain device specif stuff so maaaaany keys ...

Additionally I would love to not get them as exceptions.

I think it makes more sense to have it like https://github.com/iobroker-community-adapters/ioBroker.hmip/blob/master/main.js#L976-L979

message = Deprecation<adaptername>
with "info" as severity

and as data object we could set
* "version" of the adapter (i think makes sense)
* "deprecation": the message that contains the error string